grpc-api/src/main/java/io/deephaven/grpc_api/console/ScopeTicketResolver.java

```diff
@@ -87,17 +127,32 @@ public void forAllFlightInfo(final Consumer<Flight.FlightInfo> visitor) {
     }
 
     private static ByteString schemaBytesFromTable(final Table table) {
-        final String[] columnNames = table.getDefinition().getColumnNamesArray();
-        final ColumnSource<?>[] columnSources = table.getColumnSources().toArray(ColumnSource.ZERO_LENGTH_COLUMN_SOURCE_ARRAY);
         final FlatBufferBuilder builder = new FlatBufferBuilder();
-        builder.finish(BarrageSchemaUtil.makeSchemaPayload(builder, columnNames, columnSources, table.getAttributes()));
-        return ByteString.copyFrom(builder.dataBuffer());
+        builder.finish(BarrageSchemaUtil.makeSchemaPayload(builder, table.getDefinition(), table.getAttributes()));
+        return ByteStringAccess.wrap(builder.dataBuffer());
     }
 
-    private static Flight.Ticket ticketForName(final String name) {
-        final byte[] ticket = (TICKET_PREFIX + '/' + name).getBytes();
-        return Flight.Ticket.newBuilder()
-                .setTicket(ByteString.copyFrom(ticket))
-                .build();
+    private static String nameForTicket(final Flight.Ticket ticket) {
+        return nameForTicket(ticket.getTicket().asReadOnlyByteBuffer());
+    }
+
+    private static String nameForTicket(final ByteBuffer ticket) {
+        if (ticket.remaining() < 3 || ticket.get(0) != TICKET_PREFIX || ticket.get(1) != '/') {
+            throw GrpcUtil.statusRuntimeException(Code.FAILED_PRECONDITION, "ticket is not a ScopeTicketResolver ticket");
+        }
+        return ticket.toString().substring(2);
```

Comment:
This won't work. You need to take the bytes from 2... n-1 and make a string in presumably UTF-8 encoding.